### DISCUSSION TITLE
CNBC

### DISCUSSION THREAD

**Discussion by bugdaboss** [iptv-org/community#943] [Show and tell]
### Channel ID

CNBC.us

### Stream URL (optional)

https://1915617474.rsc.cdn77.org/TCNF6iFHMvpR6BW_vgvTBA==,1690736057/1915617474/tracks-v1a1/mono.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by kaikyoofers** [comment]
Does not work for me (tokened) @freearhey please remove the valid stream tag.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_moveonjoy.m3u#L22-L23
